docs/video/smoothie/installation.md

```diff
@@ -46,16 +46,16 @@ icon: material/folder-download
 
     1. [A portable program](https://en.wikipedia.org/wiki/Portable_application#Portable_application) means it doesn't come with an installer, you just unzip and run it. Upsides is it's easy to uninstall (just delete the folder lol) but downsides is you gotta make shortcuts to it yourself
 
-    Extract the `smoothie-rs` folder somewhere, then in it you can use `launch.cmd` to launch it in it's GUI mode
+    Extract the `smoothie-rs` folder somewhere, then in it you can use `launch.cmd` to launch it in it's GUI mode.
 
 
     # Making a [Send To](../sendto.md) shortcut and [RIFE models](./recipe.md#pre-interp)
 
-    Unzip it wherever you'd like, in the bin folder get `smoothie-rs.exe`'s PATH by <kbd>SHIFT+Right click</kbd>ing it and "Copy Path"
+    Navigate to `...\smoothie-rs\bin` and get `smoothie-rs.exe`'s PATH by <kbd>SHIFT+Right click</kbd>ing it and "Copy Path"
 
     In `%APPDATA%\Microsoft\Windows\SendTo` make a shortcut to <smoothie-rs-folder\bin\smoothie-rs.exe> with add a space and ` --tui -i`
 
-    If Smoothie is crashing on you can add the `-v` argument (that'd be ` -v --tui -i`) after the smoothie-rs executable's path to enable verbose logging to see what's wrong
```

Comment:
which executable to mention is important as it's amongst many others in /bin/

actually nvm since if people are using it from the cli they surely must be typing in the first place

but not if i add aliases so actually revert that